frontend/src/layouts/ProjectLayout/ProjectLayout.tsx

```diff
@@ -105,6 +105,20 @@ export const ProjectLayout = () => {
                           )}
                         </Link>
                       )}
+                      {isCmek && (
+                        <Link
+                          to={`/${ProjectType.KMS}/$projectId/kmip` as const}
+                          params={{
+                            projectId: currentWorkspace.id
+                          }}
+                        >
+                          {({ isActive }) => (
+                            <MenuItem isSelected={isActive} icon="lock-closed">
```

Comment:
New icon i guess? Let me know if you have any concept and I'll make one for you quickly